xbmc/cores/RetroPlayer/buffers/RenderBufferOpenGL.h

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "video/RenderBufferSysMem.h"
```

Comment:
Can you use paths relative to the root xbmc/ folder for non-same-directory files?